broken_hand_icons.txt

```diff
@@ -0,0 +1,8152 @@
+/obj/item/weapon/handcuffs/cult is missing left hand icon called "cultcuff".
+/obj/item/weapon/handcuffs/cult is missing right hand icon called "cultcuff".
```

Comment:
I don't think you intended to include this file

Reply:
whoops